src/TransactionTestsFiller/ttSignature/PointAtInfinityFiller.json

```diff
@@ -0,0 +1,21 @@
+{
+    "PointAtInfinity" :
+    {
```

Comment:
can you please add "_info" : { "comment" : "  DESCRIPTION " }
explaining a bit the math. so to know which point at infinity.